clients/go/wire.go

```diff
@@ -0,0 +1,67 @@
+package raccoon
+
+import (
+	"errors"
+
+	"encoding/json"
+
+	"google.golang.org/protobuf/proto"
+)
+
+// WireMarshaler defines a conversion between byte sequence and raccoon request payloads.
+type WireMarshaler interface {
+
+	// Marshal marshals "any" into byte sequence.
+	Marshal(any interface{}) ([]byte, error)
+
+	// Unmarshal unmarshals "data" into "any".
+	// "any" must be a pointer value.
+	Unmarshal(data []byte, any interface{}) error
+
+	// ContentType returns the Content-Type which this marshaler is responsible for.
+	ContentType() string
+}
+
+// JsonWire is a Marshaler which marshals/unmarshals into/from serialize json bytes
+type JsonWire struct{}
```

Comment:
Can we avoid Wire & Marshal totally from all the files and use simple terms so that the client can understand the interface much better. These terms are too technical IMO.

Reply:
IMO, wire is apt for this.

Reply:
> The wire implementation is not currently exposed to the user; it will be used internally by the client.
> and the the default serialiser for

"Wire" is confusing term for the client. 
Since it is not exposed to the client, I agree with you. Resolving this.